src/accelerate/commands/config/config_args.py

```diff
@@ -109,6 +110,12 @@ def from_json_file(cls, json_file=None):
             config_dict["use_cpu"] = False
         if "debug" not in config_dict:
             config_dict["debug"] = False
+        extra_keys = set(config_dict.keys()) - set(cls.__dataclass_fields__.keys())
+        if len(extra_keys) > 0:
+            raise ValueError(
+                f"Unknown keys in the config file: {list(extra_keys)}, please try upgrading your `accelerate` version or remove them."
```

Comment:
```suggestion
                f"Unknown keys in the config file: {sorted(extra_keys)}, please try upgrading your `accelerate` version or fix the keys."
```

To explain: 

1. Maybe the wrong key is just a typo, so "remove" is too specific. 
2. Explicitly say "the keys", otherwise users might think "them" refers to accelerate ;-)
3. `sorted` instead of `list` because `set` has no order, so theoretically the tests could just start failing with different Python versions. Also, maybe nicer for users to see the keys in alphabetical order.

Reply:
Went with a better option :)